sf-docker/.env.sfstation

```diff
@@ -2,7 +2,7 @@
 HOST=0.0.0.0
 
 # Port number, where SF AccessStation server will bind to
-PORT=80
+PORT=8000
```

Comment:
Not sure if this is not overriden by sfstation service definition in docker-compose.yml file. Also port mapping is probably needed to update there

Reply:
Like this? See commit 3b8820b

Reply:
yes. Also it seems to me that environment section in docker-compose can be removed

Reply:
done